lib/t_web/channels/feed_channel.ex

```diff
@@ -66,8 +65,8 @@ defmodule TWeb.FeedChannel do
 
     reply =
       %{"mode" => "live"}
-      |> Map.put("session_expiration_date", session_end)
-      |> Map.put("live_session_duration", live_session_duration)
+      |> Map.put("session_start_date", session_start)
```

Comment:
@akhrail1996 I'd keep it, otherwise it's a breaking change for the clients.

Reply:
@ruslandoga we can keep them but I planned to force-upgrade for clients anyways (since old clients do not support `voicemail` which is crucial now) -> doesn't make much sense (unless we want to release this backend prior to app release - which may be beneficial, but probably requires extra work)

for App Store review I made the app support current backend as well

Reply:
> but probably requires extra work

At least on the backend, there is no other breaking change.

Reply:
@ruslandoga yeah okay, I will put it back then (a bit of extra work is putting it back & forth into tests 😉)